core/classifier/__init__.py

```diff
@@ -1497,12 +1531,10 @@ def target_age(self, audience):
         if target_age_min is None:
             target_age_min = target_age_max
 
-        if target_age_max is None:
```

Comment:
Left this part of the code out because it doesn't make sense to set max age to min age - we end of with e.g. (17, 17).